include/deal.II/grid/tria.h

```diff
@@ -3482,6 +3482,33 @@ class Triangulation : public Subscriptor
     std::pair<std::pair<cell_iterator, unsigned int>, std::bitset<3>>> &
   get_periodic_face_map() const;
 
+  /**
+   * Translate the unique id of a coarse cell to its index.
+   *
+   * @note: For serial and shared triangulation both id and index are the same.
+   *        For distributed triangulations setting both might differ, since the
+   *        id might correspond to a global id and the index a local id.
```

Comment:
```suggestion
   *        id might correspond to a global id and the index to a local id.
```

Reply:
@masterleinad OK, I have changed it. Thanks for the review!